shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -51,6 +52,7 @@ - (instancetype)initWithFrame:(NSRect)frame
   self = [super initWithFrame:frame];
   if (self) {
     [self setWantsLayer:YES];
```

Comment:
`setBackgroundColor` already calls `setWantsLayer`. Either call it in constructor, or in setter, but I don't think it should be in both?